### PR TITLE
Reduces the meat requirement in the burrito's recipe

### PR DESCRIPTION
**Pull request by Alberyk**
-adds the following suggestion: https://forums.aurorastation.org/viewtopic.php?f=18&t=11810
-changes the vegan burrito sprite
-reduces the total protein of the burrito

### CODE REVIEW

**Comment by MarinaGryphon** [comment]
Where's the flan? :(

**Comment by JackBewt** [comment]
Changing the sprite is not necessary.

**Comment by BurgerLUA** [comment]
Yes it is. The vegan burrito has carrots and cabbage in it.

**Comment by Alberyk** [comment]
Changed the sprites to be a mere pice of tofu inside of it, reduced the protein added to the burito and removed a lore inconsistency meanwhile.

File: code/modules/food/recipes_microwave.dm [hunk 1]
```diff
@@ -1207,3 +1205,10 @@ I said no!
 /datum/recipe/mashedpotato
 	fruit = list("potato" = 1)
 	result = /obj/item/weapon/reagent_containers/food/snacks/mashedpotato
+
+/datum/recipe/icecreamsandwich
```
Comment (Arrow768) [inline code/modules/food/recipes_microwave.dm]: The icecreamsandwich is not listed in the changelog.
Also it seems the result is a porkbowl ?

**Pull request merged**